src/reducers/ast.js

```diff
@@ -18,13 +18,15 @@ import type { Record } from "../utils/makeRecord";
 export type SymbolsMap = Map<string, SymbolDeclarations>;
 
 export type ASTState = {
-  symbols: SymbolsMap
+  symbols: SymbolsMap,
+  outOfScopeLocations: ?Object
```

Comment:
The type for this would be `AstLocation[]` (even though I forgot to type the return value ... oops).